<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clevo xSM wmi Utility</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="radio_random_color">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Color</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_custom">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="labelMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Background mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_breathe">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Breathe</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_cycle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cycle</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_wave">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wave</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_dance">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dance</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_tempo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tempo</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_flash">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Flash</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioModeGroup</string>
    </attribute>
   </widget>
   <widget class="QComboBox" name="selectLeft">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Backlight colors (if Custom mode)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLeftColor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Left side</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectCenter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCenterColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectRight">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRightColor">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Right side</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectLower">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLowerColor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower front bar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonApply">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_brightness">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radioModeGroup"/>
 </buttongroups>
</ui>
